arrow-version/js/main.js

```diff
@@ -65,59 +98,69 @@ function loadEvent() {
-    arrowCoreAnimation.play();
-    incubatorCoreAnimation.play();
-    arrowBaseAnimation.stop();
+      if (screenSize() > 992) {
+        arrayArrowFeatures.map(obj => obj.classList.remove('active'));
+        corePlayHover.classList.add('active');
+        checkActiveFeature(arrayArrowFeatures);
+        arrowCoreAnimation.play();
+        incubatorCoreAnimation.play();
+        arrowBaseAnimation.stop();
+      }
   });
 
   // fx elements
   const fxPlayHover = document.getElementById('fx');
 
   fxPlayHover.addEventListener('mouseenter', () => {
-    arrayArrowFeatures.map(obj => obj.classList.remove('active'));
-    fxPlayHover.classList.add('active');
-    checkActiveFeature(arrayArrowFeatures);
-    arrowFxAnimation.play();
-    incubatorFxAnimation.play();
-    arrowBaseAnimation.stop();
+    if (screenSize() > 992) {
+      arrayArrowFeatures.map(obj => obj.classList.remove('active'));
+      fxPlayHover.classList.add('active');
+      checkActiveFeature(arrayArrowFeatures);
+      arrowFxAnimation.play();
+      incubatorFxAnimation.play();
+      arrowBaseAnimation.stop();
+    }
   });
 
   // meta elements
   const metaPlayHover = document.getElementById('meta');
 
   metaPlayHover.addEventListener('mouseenter', () => {
-    arrayArrowFeatures.map(obj => obj.classList.remove('active'));
-    metaPlayHover.classList.add('active');
-    checkActiveFeature(arrayArrowFeatures);
-    arrowMetaAnimation.play();
-    incubatorMetaAnimation.play();
-    arrowBaseAnimation.stop();
+    if (screenSize() > 992) {
+      arrayArrowFeatures.map(obj => obj.classList.remove('active'));
+      metaPlayHover.classList.add('active');
+      checkActiveFeature(arrayArrowFeatures);
+      arrowMetaAnimation.play();
+      incubatorMetaAnimation.play();
+      arrowBaseAnimation.stop();
+    }
   });
 
   // optics elements
   const opticsPlayHover = document.getElementById('optics');
 
   opticsPlayHover.addEventListener('mouseenter', () => {
-    arrayArrowFeatures.map(obj => obj.classList.remove('active'));
-    opticsPlayHover.classList.add('active');
-    checkActiveFeature(arrayArrowFeatures);
-    arrowOpticsAnimation.play();
-    incubatorOpticsAnimation.play();
-    arrowBaseAnimation.stop();
+    if (screenSize() > 992) {
+      arrayArrowFeatures.map(obj => obj.classList.remove('active'));
+      opticsPlayHover.classList.add('active');
+      checkActiveFeature(arrayArrowFeatures);
+      arrowOpticsAnimation.play();
+      incubatorOpticsAnimation.play();
+      arrowBaseAnimation.stop();
+    }
   });
 
   // incubator elements
   const incubatorPlayHover = document.getElementById('incubator');
 
   incubatorPlayHover.addEventListener('mouseenter', () => {
-    arrayArrowFeatures.map(obj => obj.classList.remove('active'));
-    incubatorPlayHover.classList.add('active');
-    checkActiveFeature(arrayArrowFeatures);
-    incubatorHoverAnimation.play();
-    arrowBaseAnimation.stop();
+    if (screenSize() > 992) {
```

Comment:
Why 992?

Reply:
Also, why not make it a constant in the file?

Reply:
992 is the break point we´ve set to hide the code console and the main logo, so that´s the point in which it has no sense to keep the main hover behaviour. At that point the mobil mode comes into play.
Anyway is a good idea make it a constant of course.
Thanks @pakoito !